src/compute-client/src/controller.rs

```diff
@@ -1062,55 +1078,12 @@ where
         ))
     }
 
-    #[mz_ore::instrument(level = "debug")]
-    fn record_introspection_updates(&mut self, storage: &mut dyn StorageController<Timestamp = T>) {
-        use IntrospectionType::*;
-
-        // We could record the contents of `introspection_rx` directly here, but to reduce the
-        // pressure on persist we spend some effort consolidating first.
-        let mut updates_by_type = BTreeMap::new();
-
-        for (type_, updates) in self.introspection_rx.try_iter() {
-            updates_by_type
-                .entry(type_)
-                .or_insert_with(Vec::new)
-                .extend(updates);
-        }
-        for updates in updates_by_type.values_mut() {
-            consolidate(updates);
```

Comment:
How useful was the consolidation here? We're removing it, but I wonder why it was here in the first place.

Reply:
The answer is I don't know! The writer task in the `CollectionManager` also consolidates, so doing it in the compute controller only affects how much data we send over the channels. I'm not sure if I was aware of that when I wrote this code originally. I'm pretty sure I didn't add it because of a performance issue I observed, but because of an abundance of caution.